lib/anycable/config.rb

```diff
@@ -122,9 +126,13 @@ def to_redis_params
 
         params[:sentinels] = sentinels.map { |sentinel| parse_sentinel(sentinel) }
       end.tap do |params|
-        next unless redis_url.match?(/rediss:\/\//) && !redis_tls_verify?
+        next unless redis_url.match?(/rediss:\/\//)
 
-        params[:ssl_params] = {verify_mode: OpenSSL::SSL::VERIFY_NONE}
+        params[:ssl_params] = {
+          verify_mode: (OpenSSL::SSL::VERIFY_NONE unless redis_tls_verify?),
+          cert: (OpenSSL::X509::Certificate.new(File.read(redis_tls_client_cert_path)) if redis_tls_client_cert_path),
```

Comment:
Should we check that both cert and key provided? Or we can use just one of them?

Also, I think, it worth making `#redis_tls_verify?` return true if the cert/key pair is configured? (Without forcing users to also specify `--redis-tls-verify`)

Reply:
> Should we check that both cert and key provided? Or we can use just one of them?

Yes, none or both must be present

> I think, it worth making #redis_tls_verify? return true if the cert/key pair is configured? 

No, it is totally unrelated. `redis_tls_verify` is about verifying server's certificate by client, while `redis_client_{cert,key}_path` is about specifying credentials to verify client by server.

Reply:
Added check for both cert and key provided